farmdata2_modules/fd2_tabs/fd2_example/fd2_example.info

```diff
@@ -4,4 +4,5 @@ core = 7.x
 package = FarmData2
 
 stylesheets[all][] = '../resources/fd2.css'
-scripts[] = '../resources/fieldsDropdownComponent.js'
\ No newline at end of file
+scripts[] = '../resources/fieldsDropdownComponent.js'
```

Comment:
The fieldsDropdownComponent should not be added to scripts since it is no longer being used.